bundles/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/internal/handler/PowermaxBridgeHandler.java

```diff
@@ -513,7 +529,7 @@ private void updateChannelsFromAlarmState(PowermaxState state) {
      * @param state: the alarm system state
      */
     private synchronized void updateChannelsFromAlarmState(String channel, PowermaxState state) {
-        if (state == null) {
+        if (state == null || !isConnected()) {
```

Comment:
Adding that could cause a problem in updateMotionSensorState to update the channel TRIPPED after a delay.
But this is solved by your code setting all channels to NULL/UNDEF in case of disconnection ?

Reply:
Yes, that was the idea!